internal/util/cgo/manager_active.go

```diff
@@ -0,0 +1,95 @@
+package cgo
+
+import (
+	"reflect"
+
+	"go.uber.org/atomic"
+)
+
+const (
+	registerIndex      = 0
+	maxSelectCase      = 65535
+	defaultRegisterBuf = 1
+)
+
+var futureManager *activeFutureManager
+
+func init() {
+	futureManager = newActiveFutureManager()
+	futureManager.Run()
+}
+
+type futureManagerStat struct {
+	ActiveCount int64
+	GCCount     int64
+}
+
+func newActiveFutureManager() *activeFutureManager {
+	manager := &activeFutureManager{
+		gcManager:     newGCManager(),
+		activeCount:   atomic.NewInt64(0),
+		activeFutures: make([]basicFuture, 0),
+		cases:         make([]reflect.SelectCase, 1),
+		register:      make(chan basicFuture, defaultRegisterBuf),
```

Comment:
Yes, consume of register chan only cost n `us` duration, can support more than 10w+ qps registration.
I will add a metric to monitoring it, so it can be seen in testing.